src/calculateRentalCost.js

```diff
@@ -1,10 +1,27 @@
 /**
  * @param {number} days
  *
+ *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRASE_OF_DAY = 40;
```

Comment:
There seems to be a typo in the constant name `PRASE_OF_DAY`. It should be `PRICE_PER_DAY` to accurately represent the daily rental cost and to fix the spelling error.